heron/healthmgr/src/java/org/apache/heron/healthmgr/policy/ToggleablePolicy.java

```diff
@@ -75,24 +75,33 @@ public ToggleablePolicy(
 
   @Override
   public Collection<Measurement> executeSensors() {
-    for (TopologyAPI.Config.KeyValue kv
-        : physicalPlanProvider.get().getTopology().getTopologyConfig().getKvsList()) {
-      LOG.fine("kv " + kv.getKey() + ":" + kv.getValue());
-      if (kv.getKey().equals(policyIdRuntime)) {
-        try {
-          PolicyMode val = PolicyMode.valueOf(kv.getValue());
-          if (!policyMode.equals(val)) {
-            policyMode = val;
-            LOG.info("policy " + policyId + " status changed to " + policyMode);
-          } else {
-            LOG.fine("policy " + policyId + " status remains same " + policyMode);
+    LOG.info("----------------------------------");
```

Comment:
is this log really needed?

Reply:
since multiple policy logs are mixed in the log loop. this helps to find when each policy begins.